lib/user_repository.rb

```diff
@@ -36,6 +55,8 @@ def user_builder(record)
     user.id = record['id'].to_i
     user.handle = record['handle']
     user.verified = record['verified']
+    user.verified = user.verified.eql?('t') ? true : false
```

Comment:
Suggestion: could add the ternary onto the end of line 57? Or would that make the line too long?

Reply:
Yes, great call! Where should this be changed? In the pull request in GitHub?